### PR TITLE
remove art_pci-dss.xml, change security_pcidss.xml to an article

### PR DESCRIPTION
**Pull request by cjschroder**
### PR creator: Description
Migrate PCI-DSS appendix from Security Guide to Unversioned Repo
Removed art_pci-dss.xml, which was for publishing  security_pcidss.xml as an article. Changed security_pcidss.xml to an article, since it no longer needs to be configured to be built as either a chapter or an article

### CODE REVIEW

**Comment by cjschroder** [comment]
@tomschr, when you have time please review this for correct structure. My next step is to proofread & make grammar and formatting corrections.

File: xml/security_pcidss.xml [hunk 1]
```diff
@@ -1,24 +1,39 @@
...
+  <productname>&productnamex;</productname>
+  <date><?dbtimestamp format="B d, Y" ?></date>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker>
+    <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
+    <dm:component>Documentation</dm:component>
+    <dm:product>SUSE Linux Enterprise Server 15 code stream</dm:product>
+    <dm:assignee>taroth@suse.com</dm:assignee>
+   </dm:bugtracker>
+   <dm:editurl>https://github.com/SUSE/doc-sle/edit/main/xml/</dm:editurl>
```
Comment (tomschr) [inline xml/security_pcidss.xml:31]: The URL should be corrected. It's not on doc-sle anymore. I'd suggest:

```suggestion
   <dm:editurl>https://github.com/SUSE/doc-unversioned/edit/main/xml/</dm:editurl>
```

**Review by tomschr — APPROVED** [review]
@cjschroder Just have one minor issue. Otherwise it looks good. Thank you!

**Comment by cjschroder** [comment]
> @cjschroder Just have one minor issue. Otherwise it looks good. Thank you!
> 
> What's related (but cannot be covered in this PR): we also need to create a new config file on docserv.

Thank you!

**Pull request merged**